tests/MVTLayer.spec.js

```diff
@@ -1,19 +1,33 @@
-import {MVTLayer} from '@/MVTLayer.js';
+import {jest} from '@jest/globals';
+// import {MVTLayer} from '@/MVTLayer.js';
 import {MVTFeature} from '@/MVTFeature.js';
-import {mockMVTSource, mockTileContext, mockVectorTileFeatures} from './common-mocks.js';
+import {mockMVTSource, mockTileContext, mockVectorTileLayers} from './common-mocks.js';
 
-// TODO jest.mock does not seem to work at all like it does in Beyond Maps and I have
-// no idea why. The MVTFeature class does _not_ get mocked by this line.
-jest.mock('@/MVTFeature.js');
-
-const mockStyle = jest.fn(() => 'style return');
+const mockStyle = {mock: 'style'};
 const mockDrawFn = jest.fn();
 
+jest.unstable_mockModule('@/MVTFeature.js', () => {
```

Comment:
I think it's markes `unstable` because it's not reeeealllyy a released feature yet, they don't have full support for mocking when your tests are ES6 modules. It seems to work here though